launch.go

```diff
@@ -440,6 +441,11 @@ func main() {
 			Usage: "Location of meta temporarily",
 			Value: "/sd/meta",
 		},
+		cli.StringFlag{
+			Name:  "store-uri",
```

Comment:
Store-uri and Store-url coexist. Does it express important difference between them?